<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LockWidget</class>
 <widget class="QWidget" name="LockWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>748</width>
    <height>433</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="btnKeyboard">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>10</y>
     <width>80</width>
     <height>26</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="widgetTime" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>280</y>
     <width>300</width>
     <height>150</height>
    </rect>
   </property>
   <widget class="QLabel" name="lblDate">
    <property name="geometry">
     <rect>
      <x>13</x>
      <y>90</y>
      <width>280</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Date</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblTime">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>11</y>
      <width>280</width>
      <height>60</height>
     </rect>
    </property>
    <property name="text">
     <string>Time</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="btnSwitchUser">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>10</y>
     <width>80</width>
     <height>26</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btnPowerManager">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>10</y>
     <width>80</width>
     <height>26</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
